.mercury/docs/guides/worktree-workflow.md

```
@@ -0,0 +1,218 @@
+   git worktree remove .worktrees/{taskId}
+   git branch -d feature/TASK-{taskId}-{slug}
+   ```
+
+---
+
+## Prompt Integration
+
+`buildDevPrompt` should prepend a worktree constraint block when
+`task.worktreePath` is set:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)